daemon/container_windows.go

```diff
@@ -38,6 +39,16 @@ func (container *Container) initializeNetworking() error {
 	return nil
 }
 
+// ConnectToNetwork connects a container to the network
+func (container *Container) ConnectToNetwork(idOrName string) error {
```

Comment:
WIndows containers have a different network model and it is not using the CNM model yet.
Hence dynamic network (dis)connection is not fully thought through. There is a separate effort
underway to bring windows closely aligned with CNM. this must be taken up along with that effort.